doc/source/changes/version_0_26.rst.inc

```diff
@@ -280,6 +297,22 @@ Miscellaneous improvements
      b [2]: 'b0' 'b1'
     dtype: float64
 
+* To create a 1D array using from_string() or read_csv() and the default separator ' ', a `nan`, `NaN` or a tabulation
+  character `\t` must be used in front of the data line:
+
+    >>> from_string('''sex  M  F
+    ...                \t   0  1''')
+    sex  M  F
+         0  1
+    >>> from_string('''sex  M  F
+    ...                nan  0  1''')
+    sex  M  F
+         0  1
+    >>> from_string('''sex  M  F
```

Comment:
I am already a bit sceptical about showing the nan example, but having both NaN and nan examples seems really overkill/ redundant with the fact you already tell you can use either, especially for a feature (from_string) that I think none of our users ever used. Mentioning read_csv here will confuse our users more than anything I think. As far as I understand, this only applies to read_csv when using ' ' as the separator, which nobody ever does.